bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/oven.xml

```diff
@@ -22,10 +22,22 @@
 			<channel id="duration" typeId="duration"/>
 			<channel id="elapsed" typeId="elapsed"/>
 			<channel id="finish" typeId="finish"/>
-			<channel id="target" typeId="target"/>
-			<channel id="measured" typeId="measured"/>
-			<channel id="temp1" typeId="temp"/>
-			<channel id="temp2" typeId="temp"/>
+			<channel id="target" typeId="temperature">
+				<label>Target Temperature</label>
+				<description>Target temperature to be reached by the oven</description>
+			</channel>
+			<channel id="measured" typeId="temperature">
+				<label>Measured Temperature</label>
+				<description>Actual measured temperature in the oven</description>
+			</channel>
+			<channel id="temp1" typeId="temperature">
+				<label>Temperature</label>
+				<description>Program temperature in the oven</description>
+			</channel>
+			<channel id="temp2" typeId="temperature">
+				<label>Temperature</label>
+				<description>Program temperature in the oven</description>
+			</channel>
```

Comment:
Same remark

Reply:
You convinced me to split into current and target for reuse of same label - fixed.